### ISSUE TITLE
Is it responsive? or will it be?

### ISSUE THREAD

**Issue by walterbaidal** [skywalkapps/bootstrap-notifications#5]
I tried opening it on my mobile phone and it wasn't responsive

**Comment by skywalkapps** [comment]
Could you be more specific? Give me an example

**Comment by walterbaidal** [comment]
Yeah, sorry, this is what I was trying to say: https://i.gyazo.com/bb63a8240fe3a963f300a728d2cc444c.mp4

As you can see, I used a dropdown-button from bootstrap for the user options and your bootstrap-notifications for the notificator, but in mobile phones the 'dropdown-notifications' is not responsive. It has a width of 450px in every device
